### PR TITLE
Glossary update, selection methods filled in, porting some aspects of hyphy.org

### PR DESCRIPTION
**Pull request by sjspielman**
Note, copy-editing still needed.

### CODE REVIEW

**Comment by stevenweaver** [comment]
Hi Stephanie, 
With this pull request, is it ready to undergo structural editing?

File: docs/glossary.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The Model Fits Table reports a summary of the models fit to the data.
     * **Full Model** - Refers to the full aBSREL model fit to the tree, with the number of ω categories per branch inferred using the adaptive algorithm.
 * ***log* L** - The log likelihood estimate of the respective model fit.
 * **\# par.** - The number of estimated parameters in the respective model.
-* **Time to fit** - Total clock time, in seconds, to fit the respective model.
+* **Time to fit** - Total clock time to fit the respective model.
```
Comment (stevenweaver) [inline docs/glossary.md]: We should place units in the results page itself

File: docs/selection-methods.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ HyPhy provides a suite of diverse phylogenetic methodologies for testing specifi
 * [BUSTED](selection-methods/#BUSTED) (**B**ranch-**S**ite **U**nrestricted **S**tatistical **T**est for **E**pisodic **D**iversification) will test for gene-wide selection at pre-specified lineages. This method is particularly useful for relatively small datasets (fewer than 10 taxa) where other methods may not have sufficient power to detect selection. **This method is not suitable for identifying specific sites subject to positive seleciton.**
 
 ### Has gene-wide selection pressure been relaxed or intensified along a certain subset of branches?
-* [RELAX](selection-methods/#RELAX) tests for a relaxation (e.g. where purifying selection has become less stringent) or an intensification (e.g. where purifying selection has become stronger) of selection pressures along a specified set of "test" branches. **This method is not suitable for the detection of positive selection.**
+* [RELAX](selection-methods/#RELAX) tests for a relaxation (reduction in stringency) or an intensification (increase in stringency) of selection pressures along a specified set of "test" branches. **This method is not suitable for the detection of positive selection.**
```
Comment (stevenweaver) [inline docs/selection-methods.md]: Is there a reason why we're using the word "stringency" as opposed to the entire sentence?

Comment (sjspielman) [reply docs/selection-methods.md]: No reason, hence the need for copy editing! And yes, structural editing a go. Also note, turns out that time units are already in the results page (at least in the next version of vision) which is why I deleted from glossary.

**Pull request merged**